src/components/camera.js

```diff
@@ -43,6 +43,7 @@ module.exports.Component = registerComponent('camera', {
     if (!oldData || oldData.active !== data.active) {
       if (data.active) {
         sceneEl.setActiveCamera(camera);
+        sceneEl.emit('active-camera-set', { camera: camera });
```

Comment:
- Can we do `camera-set-active`? Want to prefix our event names with the component emitting it.
- Can we pass in the camera element rather than the camera object to give more info?